testutil/consulhelper.go

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package testutil
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	consulapi "github.com/hashicorp/consul/api"
+	goversion "github.com/hashicorp/go-version"
+	"github.com/hashicorp/vault/sdk/helper/docker"
+)
+
+// LatestConsulVersion is the most recent version of Consul which is used unless
+// another version is specified in the test config or environment. This will
+// probably go stale as we don't always update it on every release but we rarely
+// rely on specific new Consul functionality so that's probably not a problem.
+const LatestConsulVersion = "1.15.3"
```

Comment:
Is there any reason we can't use the [latest](https://hub.docker.com/layers/hashicorp/consul/latest/images/sha256-25a8659bdf14ff9da45ce38dfacd94cb868633b60fe7291a92d657ff7edd9ba9?context=explore) tag instead of hardcoding it?

Reply:
Good question, I'm guessing this was initially done in Vault for a stable version pin? In any case, I updated the value to be `latest` and will confirm things are good in CI 👍🏼